cmd/proxy.go

```diff
@@ -33,6 +33,8 @@ var proxyFlags = struct {
 	ignoreDuplicateRequests   bool
 	recordOnlyResponseHeaders []string
 	flatResponseFileStructure bool
+	soap11Mode                bool
+	insecure                  bool
```

Comment:
Let's name this `insecureHttps` with flag `--insecure-https` to clarify what we mean by insecure.